pulp_2to3_migration/app/plugin/content.py

```diff
@@ -120,9 +122,16 @@ async def create_artifact(self, pulp2_storage_path, expected_digests={}, expecte
             artifact.size = expected_size
             return artifact
 
-        artifact = Artifact.init_and_validate(pulp2_storage_path,
-                                              expected_digests=expected_digests,
-                                              expected_size=expected_size)
+        try:
+            artifact = Artifact.init_and_validate(pulp2_storage_path,
+                                                  expected_digests=expected_digests,
+                                                  expected_size=expected_size)
+        except (DigestValidationError, FileNotFoundError, SizeValidationError):
+            if self.skip_corrupted:
+                _logger.warn(f'The content located in {pulp2_storage_path} is missing or '
+                             f'corrupted. It was skipped during Pulp 2to3 migration.')
```

Comment:
@jlsherrill , let me know if you have any preference for the phrasing or for the log level.